### PR TITLE
Telemetry duration

### PR DESCRIPTION
**Pull request by eliecharra**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | no
| 🚀 New feature?   | no
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | #442 
| ❓ Documentation  | yes

### CODE REVIEW

File: pkg/telemetry/telemetry.go [hunk 1]
```diff
@@ -31,6 +32,7 @@ func SendTelemetry(analysis *analyser.Analysis) {
 		Arch:           runtime.GOARCH,
 		TotalResources: analysis.Summary().TotalResources,
 		TotalManaged:   analysis.Summary().TotalManaged,
+		Duration:       uint(analysis.Duration.Seconds() + 0.5),
```
Comment (sundowndev) [inline pkg/telemetry/telemetry.go:35]: Why not using the float64 type here ?

Comment (eliecharra) [reply pkg/telemetry/telemetry.go:35]: On the backend we expect to retrieve an `uint`

Comment (wbeuil) [reply pkg/telemetry/telemetry.go:35]: Why adding 0.5 here ?

Comment (eliecharra) [reply pkg/telemetry/telemetry.go:35]: This is an elegant way to round to the nearest integer. `1.4` will become `1.9` and casted to `uint 1` and `1.5` will become `2.0` casted to `uint 2`

**Review by wbeuil — APPROVED** [review]

**Review by sundowndev — APPROVED** [review]

**Pull request merged**